templates/components/image.build.hbs

```diff
@@ -3,10 +3,22 @@
 <div v-if="readonly" class="fileUpload" :class="{ 'fileUpload-padding-top': value.length }">
   <div class="row">
     <div v-for="(image, index) in value" @click="onImageClick(index)" class="multiple-images-item">
-      <div class="image-item" :style="{ 'background-image': 'url(' + image + ')' }"></div>
+      <div
+        class="image-item"
+        :style="{ 'background-image': 'url(' + Fliplet.Media.authenticate(image) + ')' }"
```

Comment:
Adding `Fliplet.Media.authenticate()` is the main change here. The rest are formatting changes.